AsyncDisplayKit/ASTableView.mm

```diff
@@ -869,8 +920,13 @@ - (void)rangeController:(ASRangeController *)rangeController didDeleteNodes:(NSA
   }
 
   BOOL preventAnimation = animationOptions == UITableViewRowAnimationNone;
-  ASPerformBlockWithoutAnimation(preventAnimation, ^{
+  ASPerformBlockWithoutAnimationCompletion(preventAnimation, ^{
     [super deleteRowsAtIndexPaths:indexPaths withRowAnimation:(UITableViewRowAnimation)animationOptions];
```

Comment:
insert and delete are certainly the most important, but it is definitely possible for a move operation or even a reload operation to resulted in an effective changing the distance to the bottom of the content and require a batch fetch.

It is also necessary to check for changes to the sections. Is there any override point that exists where we can check for batch fetching after any form of editing? I think it should be relatively efficient to do so, and so more calls are better than fewer in this case, but we really should have a way to avoid putting the call manually in each method

If there is no other override point, we should rename the no-animation function similarly to how are you have done here, but make it specific to the table or collection. Then in that function, used for all of the editing operations in these positions, we can put the batch fetch call and also call the block inside the no animation API if the provided argument indicates that we don't want animation.